configs/templates/skylab-3.0.0/spack.yaml

```diff
@@ -0,0 +1,95 @@
+spack:
```

Comment:
Note to self. Delete this file after CI tests completed (this is not used in CI, and belongs only to the release branch)